wire-ios/Wire-iOS/Sources/UserInterface/Components/Controllers/SplitViewController.swift

```diff
@@ -240,6 +240,9 @@ final class SplitViewController: UIViewController, SplitLayoutObservable {
             return
         }
 
+        // without this line on iPad the text is displayed next to the tab item icon
+        newLeftViewController.map { setOverrideTraitCollection(.init(horizontalSizeClass: .compact), forChild: $0) }
```

Comment:
Looks like a workaround.. 😬 should we make a ticket for the future to fix this?

Reply:
It is, the split view controller should be removed. We can work on it within ticket https://wearezeta.atlassian.net/browse/WPB-8778.